settings.gradle

```diff
@@ -30,6 +8,7 @@ include 'distributions'
 include 'samples'
 include 'test:performance'
 include 'test:webapps:bookstore-spring'
+include 'management'
```

Comment:
As this module requires an external dependency, if this module is not required for all builds please condition is as done with collectd below